cypress/integration/add-delete-collection.ts

```diff
@@ -11,9 +11,14 @@ it('can add cards to the collection', () => {
   //navigate to 'Search' page
   cy.findByRole('link', { name: 'Search' }).click()
 
+  //search for 'charmander, give the debounce a chance to execute
+  cy.findAllByPlaceholderText('Card Name').type('charmander')
```

Comment:
```suggestion
  cy.findByPlaceholderText('Card Name').type('charmander')
```

there should only be one element with placeholder text